modules/launcher/src/main/java/org/mule/module/launcher/coreextension/DefaultMuleCoreExtensionManager.java

```diff
@@ -154,4 +170,9 @@ public void setPluginClassLoaderManager(PluginClassLoaderManager pluginClassLoad
     {
         this.pluginClassLoaderManager = pluginClassLoaderManager;
     }
+
+    protected ArtifactDeploymentListenerAdapter getArtifactDeploymentListenerAdapter (ArtifactDeploymentListener artifactDeploymentListener)
```

Comment:
Does this have to be protected?

Reply:
Yes, it is necessary for testing purposes.